File: acm-java/src/test/java/leetcode/LeetCode2433Test.java
package leetcode;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class LeetCode2433Test {
    LeetCode2433 sut = new LeetCode2433();

    @Test
    void testCaseOne() {
        assertArrayEquals(
            new int[]{5, 7, 2, 3, 2},
            sut.findArray(new int[]{5, 2, 0, 3, 1})
        );
    }

    @Test
    void testCaseTwo() {
        assertArrayEquals(
            new int[]{13},
            sut.findArray(new int[]{13})
        );
    }
}

File: acm-kotlin/src/test/kotlin/leetcode/LeetCode2433Test.kt
package leetcode

import org.junit.jupiter.api.Assertions.*
import org.junit.jupiter.api.Test

class LeetCode2433Test {
    private val sut = LeetCode2433()
    @Test
    fun testCaseOne() {
        assertArrayEquals(
            intArrayOf(5, 7, 2, 3, 2),
            sut.findArray(intArrayOf(5, 2, 0, 3, 1))
        )
    }

    @Test
    fun testCaseTwo() {
        assertArrayEquals(
            intArrayOf(13),
            sut.findArray(intArrayOf(13))
        )
    }
}